src/WebSockets.jl

```diff
@@ -220,7 +220,7 @@ function Base.close(ws::WebSocket; statuscode::Union{Int, Nothing}=nothing)
         closewrite(ws; statuscode=statuscode)
     end
     while !eof(ws) # FIXME Timeout in case other end does not send CLOSE?
-        readframe(ws)
+        readframe(ws; strict=false)
```

Comment:
It think it would be better to use try/catch instead of adding a flag to `readframe`:
```julia
try
    readframe(ws)
catch e
    e isa WebSocketError || rethrow(e)
end
```